guides/transactions/long-running-transactions/README.md

```diff
@@ -187,6 +189,132 @@ partial class MailPage : Json, IBound<Mail>
 
 You can find an example of this in [step 6](../../tutorial/cancel-and-delete) of the tutorial.
 
+## Mixing Transactions
+
+Transactions can be mixed as outer and inner transactions - one transaction wraps around the other. These are the possible combinations and their effects:
+
+| Outer | Inner | Effect                                  |
+|-------|-------|-----------------------------------------|
+| Long  | Long  | Execute inner as a part of outer        |
+| Long  | Short | Execute inner as a separate transaction |
+| Short | Long  | Not supported. Run-time error           |
+| Short | Short | Execute inner as part of outer          |
+
+### Long-Running in Long-Running
+
+With a long-running transaction inside a long-running transaction, they act as if they were one transaction:
+
+```cs
+[Database]
+public class Person {}
+
+[Database]
+public class Animal {}
+
+class Program
+{
+    static void Main()
+    {
+        Db.Scope(() =>
+        {
+            new Person();
+
+            Db.Scope(() =>
+            {
+                Transaction.Current.Commit(); // Commits the Person
+                new Animal();
+            });
+
+            Transaction.Current.Commit(); // Commits the Animal
+        });
```

Comment:
I didn't get it at first sight :) I know that this example is correct, but isn't it better to do it like (pseudocode):
```
DB.Scope(() => {
    new Person()
   Db.Scope(){
       new Animal()
       Transaction.Current.Rollback(); //Rollbacks both adding Person and Animal
   }
});
```
Just giving another approach, please decide as you like.